assets/js/src/subscribers/form.jsx

```diff
@@ -11,23 +11,23 @@ const fields = [
     label: MailPoet.I18n.t('email'),
     type: 'text',
     disabled: function disabled(subscriber) {
-      return Number(subscriber.wp_user_id > 0);
+      return Number(subscriber.wp_user_id > 0) || Number(subscriber.is_woocommerce_user) === 1;
```

Comment:
Isn't it better to type this to boolean (in all places)?

Reply:
This property is stored as `"1"` or `"0"` in the database and JavaScript converts `"0"` to `true` when casting to boolean. So this would require more conversions to get it right.